nixos/modules/services/audio/navidrome.nix

```diff
@@ -46,6 +46,7 @@ in {
         RuntimeDirectory = "navidrome";
         RootDirectory = "/run/navidrome";
         ReadWritePaths = "";
+        BindPaths = [] ++ lib.optional (cfg.settings ? DataFolder) cfg.settings.DataFolder;
```

Comment:
```suggestion
        BindPaths = lib.optionals (cfg.settings ? DataFolder) cfg.settings.DataFolder;
```

Reply:
I don't think this change was made?

Reply:
Sorry you are absolutely right. I made the commit without staging my changes :man_facepalming: